### PR TITLE
#159467252 Make the login page mobile responsive (update)

### PR DESCRIPTION
**Pull request by Spectrumsun**
#### What does this PR do?
- Make sure the login page is responsive
- Make sure a grid system is used

#### Description of Task to be completed?
- When the user visits the site the login page should adjust to the display of the display
- The login page should be responsive
- The login page should use a grid system

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Install project dependencies run `yarn install`
- Run the server `make start`
- To be able to login, add a supported domain name to your hosts:
  - `vim /etc/hosts` and add the line `127.0.0.1        local.andela.com`
- Open the browser and navigate to the Requests URL `http://local.andela.com:3000`

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#159467252](https://www.pivotaltracker.com/n/projects/2184887/stories/159467324)

#### Screenshots (if appropriate)
screen shot 2018-08-14 at 11 01 55 2
image

#### Questions:
None

### CODE REVIEW

File: src/views/Login/Login.scss [hunk 1]
```diff
@@ -36,7 +36,7 @@
     width: 252px;
     border-radius: 4px;
     background-color: #3359db !important;
-    margin-left: 80px;
+    margin: 0px 0px 250px 80px
```
Comment (albert85) [inline src/views/Login/Login.scss]: you don't end this line with a semi colon

**Comment by Spectrumsun** [comment]
fixed @albert85

**Comment by Spectrumsun** [comment]
@samuelkubai have aligned the logo and fix the pr message

**Comment by samuelkubai** [comment]
@Spectrumsun please use the `make` command in the section

> How should this be manually tested?

**Comment by Spectrumsun** [comment]
@samuelkubai  have add the make start command to the PR

**Pull request merged**